.github/workflows/go-test.yml

```diff
@@ -81,4 +81,13 @@ jobs:
       - name: Run smoke test
         run: |
           cd smoke_test
-          ./test.sh
\ No newline at end of file
+          ./test.sh
+
+      - name: Upload lib/l10n/ artifact
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: smoke-test-l10n-dir
+          path: |
+            smoke_test/lib/l10n/
+            !smoke_test/lib/l10n/.gitignore
```

Comment:
sorry for being pedantic

`"files.insertFinalNewline": true,` in VS Code's `settings.json` :)